test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+const data = require('./data.json');
+
+const url = 'https://api.github.com/users';
+const userName = 'aperdomob';
+
+chai.use(chaiSubset);
+
+describe('Github Api Repositories Test', () => {
+  let responseData;
+  let repoName;
+  it('GET test', async () => {
+    const response = await axios.get(`${url}/${userName}`);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal(data.name);
+    expect(response.data.location).to.equal(data.location);
+    expect(response.data.company).to.equal(data.company);
+    responseData = response;
+  });
+
+  it('search repository', async () => {
+    const responseTwo = await axios.get(`${responseData.data.repos_url}`);
+    const repository = responseTwo.data.find((repo) => repo.name === 'jasmine-json-report');
```

Comment:
instead of declaring repository object here, you can assign directly to the `jasmineRepo` variable (currently named as repoName)